android/app/build.gradle

```diff
@@ -43,18 +33,36 @@ android {
         main.java.srcDirs += 'src/main/kotlin'
     }
 
+    tasks.whenTaskAdded { task ->
+        if (task.name == 'compressReleaseAssets') {
+            task.dependsOn 'copyFlutterAssetsRelease'
+        }
+    }
+
     defaultConfig {
         applicationId "io.customer.amiapp_flutter"
         minSdkVersion 21
         targetSdkVersion 33
-        versionCode flutterVersionCode.toInteger()
-        versionName flutterVersionName
+        versionCode 1
+        versionName "1.0"
+    }
+
+    signingConfigs {
+        debug {
+            storeFile file('debug.keystore')
```

Comment:
I see file `android/app/debug.keystore` got added in this PR. Should it have? Or should that file have been added to the gitignore?